docs/spark-procedures.md

```diff
@@ -721,14 +721,15 @@ Creates a view that contains the changes from a given table.
-|---------------|----------|------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `table`       | ✔️ | string | Name of the source table for the changelog                                                                                                                                                                            |
-| `changelog_view`        |   | string | Name of the view to create                                                                                                                                                                                            |
-| `options`     |   | map<string, string> | A map of Spark read options to use                                                                                                                                                                                    |
-|`compute_updates`| | boolean | Whether to compute pre/post update images (see below for more information). Defaults to false.                                                                                                                        | 
-|`identifier_columns`| | array<string> | The list of identifier columns to compute updates. If the argument `compute_updates` is set to true and `identifier_columns` are not provided, the table’s current identifier fields will be used to compute updates. |
-|`remove_carryovers`| | boolean | Whether to remove carry-over rows (see below for more information). Defaults to true.                                                                                                                                 |
+| Argument Name        | Required? | Type                | Description                                                                                                                                                                                                                                                                                   |
+|----------------------|-----------|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `table`              | ✔️        | string              | Name of the source table for the changelog                                                                                                                                                                                                                                                    |
```

Comment:
Does it need one more space, or is it just rendering?

Reply:
I assume it is how browser renders it. It should be better now since we shortened the description.